trusted-firmware-m/config/config_default.cmake

```diff
@@ -132,6 +132,7 @@ set(ATTEST_INCLUDE_OPTIONAL_CLAIMS      ON          CACHE BOOL      "Include opt
 set(ATTEST_INCLUDE_COSE_KEY_ID          OFF         CACHE BOOL      "Include COSE key-id in initial attestation token")
 
 set(TFM_PARTITION_PLATFORM              ON          CACHE BOOL      "Enable Platform partition")
+set(PLATFORM_NV_COUNTER_MODULE_DISABLED FALSE       CACHE BOOL      "Disable Non-volatile counter module")
```

Comment:
I don't personally like the _DISABLED negative polarity. I realize it's convention though. Take it as you will :D

Reply:
Hehe, I agree. But sticking with convention wins here IMO